tests/e2e/e2e_skip.go

```diff
@@ -0,0 +1,17 @@
+package e2e
```

Comment:
Shouldn't we move this to `utils` package ?

Reply:
or at least rename file `utils.go` ?

Reply:
I am thinking where could be a better place to put this, under `utils/exec` so we use it like `exec.SkipTest()`?